tests/test_flow/test_reload.py

```diff
@@ -0,0 +1,257 @@
+import pytest
+
+import sys
+from textwrap import dedent
+
+from bionic.exception import CodeVersioningError
+
+
+# Pytest rewrites modules and caches the compiled pyc file. The cache can
+# interfere with the tests below. Hence, we disable the cache here.
+@pytest.fixture(autouse=True)
```

Comment:
Not that it matters much, but I'm curious why you used `autouse` instead of just making `dont_write_bytecode` a dependency of `tmp_module_path`?

Reply:
Moving the contents of this method to `write_module`, where it belongs.